docs/static/releasenotes.asciidoc

```diff
@@ -1,73 +1,56 @@
 [[releasenotes]]
 == Release Notes
 
-This section summarizes the changes in each release.
-
-* <<RC1, Logstash 5.0.0-rc1>>
-* <<beta1, Logstash 5.0.0-beta1>>
-* <<alpha5,Logstash 5.0.0-alpha5>>
-* <<alpha4,Logstash 5.0.0-alpha4>>
-* <<alpha3,Logstash 5.0.0-alpha3>>
-* <<alpha2,Logstash 5.0.0-alpha2>>
-* <<alpha1,Logstash 5.0.0-alpha1>>
-
-[[RC1]]
-=== Logstash 5.0.0-rc1 Release Notes
-
-* Windows: Fixed signal HUP warnings on startup ({lsissue}5239[Issue 5239]).
-* Windows: Enabled logging on Windows by fixing log4j config location ({lsissue}5971[Issue 5971]).
-* Configuration errors are now logged to file, not printed to stderr ({lsissue}5975[Issue 5975]).
-* Correctly calculate pipeline `duration_in_millis` metric ({lsissue}5952[Issue 5952]).
+This section summarizes the changes in {version} release.
 
 [float]
-==== Output Plugins
-
-*`Elasticsearch`*:
-
-* Passwords in Logstash logs partially visible when connection to Elasticsearch fails (https://github.com/logstash-plugins/logstash-output-elasticsearch/issues/482[Issue 482]).
+==== General
 
-[[beta1]]
-=== Logstash 5.0-beta1 Release Notes
-
-* Migrated Logstash's internal logging framework to Log4j2. This enhancement provides the following features:
+* A new monitoring feature provides runtime visibility into the Logstash pipeline and its plugins. This component 
+collects various kinds of operational metrics while Logstash processes your data, and all of this information 
+can be queried using simple APIs. Please refer <<monitoring>> for details.
+* Improved throughput performance across the board (up by 2x in some configs) by implementing Event 
+representation in Java. Event is the main object that encapsulates data as it flows through 
+Logstash and provides APIs for the plugins to perform processing. This change also enables 
+faster serialization for future persistence work ({lsissue}4191[Issue 4191]).
+* Breaking Change: Renamed filenames for Debian and RPM artifacts to match Elasticsearch's naming scheme. The metadata is 
+still the same, so upgrades will not be affected. Please refer to the new <<deb-layout, directory structure>>. If you have 
+automated downloads for Logstash, please make sure you use the updated URLs ({lsissue}5100[Issue 5100]).
+* Breaking Change: Introduced a new way to configure application settings for Logstash through a `logstash.yml` file. This
```

Comment:
A new feature (settings file) shouldn't be listed as a breaking change. There was no prior feature that this new feature removed.

Possibly to note for a breaking change is the fact that command line flags were renamed to match with yaml settings (--foo.bar, etc).